backend/backend.go

```diff
@@ -420,7 +432,7 @@ func (backend *Backend) createAndAddBTCAccount(
 			case signing.ScriptTypeP2WPKH:
 				suffixedName += ": bech32"
 			}
-			err := backend.CreateAndAddAccount(
+			err := backend.createAndAddAccount(
```

Comment:
Please add it to `createAndAddBTCAccount` too that it requires the accountsLock.

Reply:
done